src/table/src/requests.rs

```diff
@@ -93,8 +96,20 @@ impl TryFrom<&HashMap<String, String>> for TableOptions {
                 .into();
             options.ttl = Some(ttl_value);
         }
+        if let Some(compaction_time_window) = value.get(COMPACTION_TIME_WINDOW_KEY) {
```

Comment:
suggestion: Maybe we should return an error instead silently skip the user-specified compaction time window when fails to parse the time window value.

Reply:
Thanks, good idea, I'll try fixing it today!
Will also take a look at the conflicts.

Reply:
> Thanks, good idea, I'll try fixing it today! Will also take a look at the conflicts.

Don't worry, I will fix the conflicts.